tests/integration_tests/build/test_unittests.py

```diff
@@ -12,7 +13,7 @@
 # run coverage with the `gnu` toolchains and run unit tests with the `musl` toolchains.
 TARGET = "{}-unknown-linux-musl".format(MACHINE)
 
-
+@pytest.mark.timeout(600)
```

Comment:
What were the times, before and after?

Reply:
I  think this PR should also test the nightly perf tests.

Reply:
I think if we need to investigate changes in test timeouts it indicates we should add/improve performance tests.

I don't think we should care about or use timeouts in a similar way to performance tests.

In this case I think the timeout is fairly reasonable since when running units tests to gather code coverage it already uses a timeout of 600.